oe_theme.theme

```diff
@@ -1242,11 +1242,27 @@ function oe_theme_preprocess_block__system_branding_block(&$variables) {
   $route_options = ['absolute' => TRUE];
   $variables['site_logo_href'] = Url::fromRoute('<front>', [], $route_options)->toString();
 
-  // Ensure the site logo falls back to english version if the language is
-  // not EU language.
+  // Provide correct icon for non-eu languages.
   $language_code = \Drupal::languageManager()->getCurrentLanguage()->getId();
   if (!EuropeanUnionLanguages::hasLanguage($language_code)) {
-    $variables['current_language_id'] = 'en';
+    $language_icon_mapping = [
+      'nb' => 'no',
+      'tr' => 'tu',
+    ];
+    if (isset($language_icon_mapping[$language_code])) {
+      // Set correct icon for Norwegian and Turkish languages.
+      $variables['current_language_id'] = $language_icon_mapping[$language_code];
+    }
+    else {
+      // Check whether ECL provides a logo for the language.
+      $ecl_logo = DRUPAL_ROOT . '/' . drupal_get_path('theme', 'oe_theme') . '/dist/' . $variables['ecl_component_library'] . '/images/logo/';
+      $ecl_logo .= $variables['ecl_component_library'] === 'eu' ? 'standard-version/positive/' : '';
```

Comment:
No need for this. Check oe_theme_preprocess() function that already builds the correct logo path.

Reply:
oe_theme_preprocess build path based on docroot (/build/dist/...), here we need absolute path (/var/www/.../build), so need to build again.